docs/DEVELOPING.md

```diff
@@ -84,33 +84,28 @@ You can play with it from the parachain client, make transfers, inspect events,
 ## Linting
 
 ### Source code
-Lint the source code with `cargo +nightly fmt`. This excludes certain paths (defined in `rustfmt.toml`) that we want to stay as close as possible to `paritytech/substrate` to simplify upgrading to new releases.
+Lint the source code with `cargo fmt --all`. This excludes certain paths (defined in `rustfmt.toml`) that we want to stay as close as possible to `paritytech/substrate` to simplify upgrading to new releases.
```

Comment:
Shouldn't this be the same command that we use in our CI - `cargo fmt -- --check`?

Reply:
Good point, I actually believe we should instead change the CI to check with `--all` 😅. IMO `fmt --all` here has no downsides as it is a superset of `fmt`. I also have to admit that I am not sure whether there is a difference between `cargo fmt` and `cargo fmt --all` for our workspace.